htdocs/core/lib/functions2.lib.php

```diff
@@ -1278,10 +1278,12 @@ function get_next_value($db, $mask, $table, $field, $where = '', $objsoc = '', $
 
 	// Get counter in database
 	$counter = 0;
-	$sql = "SELECT MAX(".$sqlstring.") as val";
+
+	$sql = "SELECT ".$sqlstring." as val";
```

Comment:
If we have 1 millions of record, this may return 1 million of lines that then will be parsed line by line.

Reply:
I already try to do this in the commit 053fb76 but Travis CI failed.

Reply:
@eldy What can I do for this PR ?